src/commands/react_native/xcode.rs

```diff
@@ -263,16 +271,24 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         // With that we we then have all the information we need to invoke the
         // upload process.
         } else {
-            let rv = process::Command::new(&script)
+            let mut command= process::Command::new(&script);
```

Comment:
Please run `cargo fmt` on the project as well.

Reply:
I did, but no changes emitted.

I fixed one extra empty line manually.

Reply:
Hm, that's odd. It doesn't do anything for me either, but my editor formats this correctly. Do you mind if I push my formatting changes?

Reply:
I don't mind. Go ahead.